.mockery.yaml

```diff
@@ -27,6 +27,7 @@ dir: "{{.InterfaceDir}}/mocks"
 mockname: "{{.InterfaceNameCamel}}"
 filename: "{{.InterfaceNameSnake}}.mock.go"
 outpkg: "mocks"
+resolve-type-alias: False # see https://vektra.github.io/mockery/latest/deprecations/#resolve-type-alias
```

Comment:
This seems to me the bare minimum to appease the `make generate` command. However we should consider if we need to add
```
issue-845-fix: True # see https://vektra.github.io/mockery/latest/deprecations/#issue-845-fix
```
too

Reply:
We don't have `inpackage` parameter I don't think we need fix for issue-845 ? The issue arises only when inpackage is set to true.

> mockery ignored the outpkg: parameter if inpackage: was set to True

Reply:
I ran `make generate` with this fix and it outputs multiple warnings that we should set the issue-845-fix to True. When I set that option the warnings went away so we probably should add that too?